### PR TITLE
final project proposal

### PR DESCRIPTION
**Pull request by heidiflammable**

### CODE REVIEW

**Comment by jamdon** [comment]
I think your idea will lend itself well to meeting the required elements of the project.

**Comment by heidiflammable** [comment]
first commit of files. working but unfinished. 
still trying to think through how to get genre outputs to show up in results box. Then will work on models.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+var app = express();
+
+app.use(express.static('public'));
+
+//returns list of all bands
+app.use('/bands', bandRouter);
+
+//returns list of all genres
+app.get('/genres', function(request, response){
```
Comment (jamdon) [inline index.js:23]: if you have more than one `/genre` route, I'd think of creating a router for it

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+		</div>
+
+		<div id="band-output"; style="background-color:#c6c6c6; height:200px; overflow:hidden;">
+		   	<p>results of query go here</p>
+		   	  <ul class="filter-bands-by-genre-list"></ul>
+
+		</div>
+	</div><!--end container -->
+
+<script src="client.js"></script>
```
Comment (jamdon) [inline public/index.html]: You'll probably want your client.js to come after your jQuery script tag, especially if you're using jQuery in it.

**Comment by jamdon** [comment]
thanks, Heidi! 👍

**Pull request closed without merging**